samples/src/components/Samples/AtMountTimeout/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { PropTypes } from 'react'
+import animate from 'hoc-react-animate'
+
+import styles from '../animate.scss'
+
+const AtMount = ({ ex, className }) => (
+  <div className={`${styles.component} ${className}`}>
+    I'm animate at mount.<br />
```

Comment:
Du coup, j'écrirai plutôt `I animate at mount`
ou alors comme sur `Animate.css` => `Animate it at mount` (cf [site](https://daneden.github.io/animate.css/))